network_simulator/plot_tool.py

```diff
@@ -0,0 +1,300 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+from statistics import *
+
+class PlotTool(object):
```

Comment:
It seems like most of the functions in this class are not instance functions, i.e. they are static functions. You can label these with @staticmethod before the function signature, and can get rid of the self in the arguments. Also, get rid of the __init__ function.